### PR TITLE
Refactor w1.1 handling exception on frontend

### PR DESCRIPTION
**Pull request by draganacarapic**

### CODE REVIEW

File: HealthcareSystem/PatientWebApp/wwwroot/css/style.css [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
-#search_btn {
-    width: 10%;
-    margin-left: 45%;
-    background-color: #33adff;
+.my_btn {
     color: white;
+    background-color: rgb(43, 110, 138);
+    width: 110px;
+    height: 38px;
 }
```
Comment (vlaksi) [inline HealthcareSystem/PatientWebApp/wwwroot/css/style.css]: ### Comment

We are a company and we should have some documentation. This 'my_btn' is what? Where is it used, and why does it look how it looks?  

### Proposal
  - think about people who will read your code after you write it
  - make a more descriptive name
  - write documentation above
  - Loke at something like [this](https://webdesign.tutsplus.com/articles/css-documentation-best-practices--cms-30139)

File: HealthcareSystem/PatientWebApp/wwwroot/html/add_feedback.html [hunk 2]
```diff
@@ -72,7 +72,7 @@
 
         <!-- Send button -->
         <div class="row">
-            <input id="search_btn" type="submit" value="Send">
+            <input class="my_btn" type="submit" value="Send" style="margin-left: 45%;">
         </div>
```
Comment (vlaksi) [inline HealthcareSystem/PatientWebApp/wwwroot/html/add_feedback.html]: ### Comment

I like how you make your code readable here! Well done. 

### Proposal

This HTML code description(comment)

```sh
 <!-- Send button -->
```

is a good practice, make more code like this!

**Review by vlaksi — CHANGES_REQUESTED** [review]
### General comment

Your team should fix the next things:
  - give more descriptive names in .css files
  - make documentation like in the green section of the following image
  - be consistent, if you write documentation for one part of code, make it for all parts of code

image

**Review by vlaksi — APPROVED** [review]
Well done, much better.

**Review by anciz — APPROVED** [review]
Good enough!

**Pull request merged**